samples/nrf_rpc/protocols_serialization/server/sample.yaml

```
@@ -55,7 +55,7 @@ tests:
       - ci_build
       - ci_samples_nrf_rpc
     extra_args: >
-      SNIPPET="ble;openthread;debug;coex;log_rpc;ci-shell"
+      SNIPPET="ble;openthread;debug;coex;log_rpc;ci;ci-shell"
```

```suggestion
    extra_args: >
      SNIPPET="ble;openthread;debug;coex;log_rpc;ci-shell"
    extra_conf: >
      CONFIG_NRF_RPC_UTILS_CRASH_GEN=y
```
apply to both sample files then drop every other change in PR